src/qt/pivx/send.cpp

```diff
@@ -134,9 +134,9 @@ void SendWidget::refreshAmounts()
         totalAmount = walletModel->getBalance(CoinControlDialog::coinControl) - total;
         ui->labelTitleTotalRemaining->setText(tr("Total remaining from the selected UTXO"));
     } else {
-        // Wallet's balance
-        totalAmount = walletModel->getBalance(nullptr, fDelegationsChecked) - total;
-        ui->labelTitleTotalRemaining->setText(tr("Total remaining"));
+        // Wallet's unlocked balance
+        totalAmount = walletModel->getUnlockedBalance(nullptr, fDelegationsChecked) - total;
+        ui->labelTitleTotalRemaining->setText(tr("Unlocked remaining"));
```

Comment:
Would be better to leave it as "total" and add a tooltip with the further information about it. The "unlocked" word is not used anywhere in the GUI to describe any type of balance, users will not understand the connection right away like us.

Reply:
"Unlocked" is used in coin control for example. 
This, in my opinion, should be either "Unlocked" or "Available" or "Spendable" (without tooltips), but "Total" is definitely misleading (and redundant, as the total wallet's balance is already in the top bar).

Reply:
was thinking on available too but the space there is not really big, matter of trying it. It's better, yeah.